test/generators/typescript/TypeScriptGenerator.spec.ts

```diff
@@ -154,6 +163,7 @@ ${content}`;
   members?: string | number | boolean;
   tupleType?: [string, number];
   arrayType: Array<string>;
+  additionalProperties?: Map<String, object | string | number | Array<unknown> | boolean | null | number>;
```

Comment:
LOL, we should introduce the `any` type, really 😂

Reply:
For TS we can add here https://github.com/asyncapi/modelina/blob/master/src/generators/typescript/TypeScriptRenderer.ts#L22 logic to check this 7 types and then render `any`

Reply:
agreed we should 👍

Reply:
But we have an issue for it #198